docs/config.md

```diff
@@ -0,0 +1,19 @@
+# Config
+Some checks in ZetaSQL linter has configurable options. Users can provide a proto-txt file in order to change these options as well as disabling some linter checks.
```

Comment:
...has configurable options which can be specified in the a file contains config proto in text format.

Reply:
Changed